README.md

```diff
@@ -24,6 +24,9 @@ There are two main categories of samples: **Camera** and **Applications**. The s
     - [**GetCameraIntrinsics**][GetCameraIntrinsics-url] - Read intrinsic parameters from the Zivid camera.
     - [**FirmwareUpdater**][FirmwareUpdater-url] - Update firmware on the Zivid camera.
     - [**ZividBenchmark**][ZividBenchmark-url] - Benchmark the Zivid camera.
+  - **Maintenance**
+    - [**VerifyCameraTrueness**][VerifyCameraTrueness-url] - Check the dimension trueness of a Zivid camera.
```

Comment:
As @ajlandau suggested, perhaps we can remove 'Trueness' from the name? It may be a bit vague then. Perhaps add InField after it, so: "VerifyCameraInField" and "CorrectCameraInField"?

Reply:
Will do